packages/sdk/src/execute.ts

```diff
@@ -20,6 +20,8 @@ export type NearContractCall = ContractCall | ContractCall[]
 export type NearCallSigningOptions = {
   wallet?: Wallet;
   account?: Account;
+  callbackUrl?: string;
```

Comment:
NearCallSigningOptions are for methods used to sign the transaction, a callback URL should be passed as part of the NearContractCall